discord/channel.py

```diff
@@ -1059,6 +1066,13 @@ def last_message(self) -> Optional[Message]:
         """
         return self._state._get_message(self.last_message_id) if self.last_message_id else None
 
+    @property
+    def jump_url(self) -> str:
+        """:class:`str`: Returns a URL that allows the client to jump to the channel.
+        .. versionadded:: 2.0
```

Comment:
These need a newline between them.

```suggestion

        .. versionadded:: 2.0
```